packages/docusaurus-theme-classic/src/hooks/theme.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import * as React from 'react';
+
+const useTheme = defaultTheme => {
+  const [theme, setTheme] = React.useState(defaultTheme);
```

Comment:
should we use 

```js
const defaultTheme = typeof document !== 'undefined'
      ? document.querySelector('html').getAttribute('data-theme')	 : ''
```

if its undefined ?

Reply:
Updated to let the hook computes default by itself.